usage/app/controllers/UsageApi.scala

```diff
@@ -184,6 +184,23 @@ class UsageApi(auth: Authentication, usageTable: UsageTable, usageGroup: UsageGr
     )
   }}
 
+  def setDownloadUsages() = auth(parse.json) { req => {
+    val request = (req.body \ "data").validate[DownloadUsageRequest]
+    request.fold(
+      e => respondError(
+        BadRequest,
+        errorKey = "download-usage-parse-failed",
+        errorMessage = JsError.toJson(e).toString
+      ),
+      fur => {
```

Comment:
What's ... what's a fur? 😅

Reply:
Great question! Might be too late to ask @jamie-pitts the original author of the PR (a cost of us taking so long to get to it, sorry!!). It's likely to be a typo for `dur` as it's of type `DownloadUsageRequest`. A more intention revealing name would be good here, for sure, I'll do this on this PR.

Reply:
Ah, it's copy pasted from the FrontsUsageRequest handling above this, so not necessarily a typo.